File: 백준/Bronze/21638. SMS from MCHS/SMS from MCHS.cc
#include <iostream>
using namespace std;
int main(void) {
    int t1, v1, t2, v2;
    cin >> t1 >> v1;
    cin >> t2 >> v2;
    if (t2 < 0 && v2 >= 10)
        cout << "A storm warning for tomorrow! Be careful and stay home if possible!";
    else {
        if (t2 < t1)
            cout << "MCHS warns! Low temperature is expected tomorrow.";
        else {
            if (v2 > v1)
                cout << "MCHS warns! Strong wind is expected tomorrow.";
            else
                cout << "No message";
        }
    }
    return 0;
}

File: 백준/Bronze/21638. SMS from MCHS/SMS from MCHS.py
import sys
input=lambda:sys.stdin.readline().rstrip()
t1,v1=map(int,input().split())
t2,v2=map(int,input().split())
if t2<0 and v2>=10:
    print("A storm warning for tomorrow! Be careful and stay home if possible!")
else:
    if t2<t1:
        print("MCHS warns! Low temperature is expected tomorrow.")
    else:
        if v2>v1:
            print("MCHS warns! Strong wind is expected tomorrow.")
        else:
            print("No message")